File: Other Algorithms/isPalindrome.py
def is_palindrome(x):
    s = str(x)
    if s == s[::-1]:
        return True
    return False


def main():
    x = int(input())
    print(str(x) + " is palindrome: " + str(is_palindrome(x)))


if __name__ == '__main__':
    main()

File: Other Algorithms/isPalindrome/isPalindrome.js
// Un función booleana que devuelve true o false si la palabra es palindrome

function isPalindrome(str) {
     var wordTest = str.replace(/[^a-zA-Z0-9]/g,"").toUpperCase().split("");
  return wordTest.toString() === wordTest.reverse().toString() ;
}

console.log(isPalindrome("eye"));
